src/common_utils/language_utils/sentence_filter_utils.py

```diff
@@ -73,7 +75,6 @@ def extract_lemma_and_morphological_tag(self, word):
 
     def extract_lemma(self, word, response_cont=None):
```

Comment:
I'm still confused with this `response_cont` parameter. If you use boolean value it is more readable. I think it is the fifth time when I am mentioning it 😢